discussion_agents/cog/functional/critic.py

```diff
@@ -6,51 +6,68 @@
 
 from discussion_agents.cog.prompts.critic import (
     CRITIC_CRITIQUE_INSTRUCTION_HOTPOTQA,
+    CRITIC_CRITIQUE_INSTRUCTION_TRIVIAQA,
     CRITIC_INSTRUCTION_HOTPOTQA,
+    CRITIC_INSTRUCTION_TRIVIAQA,
 )
 
+BENCHMARK_PROMPTS = {
+    "hotpotqa": CRITIC_INSTRUCTION_HOTPOTQA,
+    "triviaqa": CRITIC_INSTRUCTION_TRIVIAQA,
+    # Add more mappings as necessary
+}
 
-def _build_agent_prompt(
-    question: str, examples: str, prompt: str = CRITIC_INSTRUCTION_HOTPOTQA
-) -> str:
+BENCHMARK_PROMPTS_CRITIQUE = {
+    "hotpotqa": CRITIC_CRITIQUE_INSTRUCTION_HOTPOTQA,
+    "triviaqa": CRITIC_CRITIQUE_INSTRUCTION_TRIVIAQA,
+}
+
+
+def _build_agent_prompt(question: str, examples: str, benchmark: str) -> str:
```

Comment:
change this back. whoever uses this function should have the freedom to pass in their own prompt